README.md

```diff
@@ -6,8 +6,29 @@ at any point, and most data is dumped every 12 hours, after analysis and stat st
 
 ## Installation
 
-Coming Soon
+1. Clone this repo on your server into the folder **jstats-server**
+2. Create a database called **jstats-server**
+3. Create a table with this code
+```
+CREATE TABLE IF NOT EXISTS `jos_jstats` (
```

Comment:
I'd rather we linked to the file so if we do any updates we only have to do it in one place